pkg/local-storage/member/controller/volume_migrate_task_worker.go

```diff
@@ -448,5 +449,21 @@ func generateJobName(mName string, pvcName string) string {
 	if len(pvcName) > 25 {
 		pvcName = pvcName[:25]
 	}
-	return fmt.Sprintf("%s-datacopy-%s", mName, pvcName)
+	jobName := fmt.Sprintf("%s-datacopy-%s", mName, pvcName)
```

Comment:
what does this name `%s-datacopy-%s` mean and what's releationship with `LocalVolumeMigrate` and `LocalVolume`?

maybe the name of `LocalVoumeMigrate` is enough for the datacopy job?